docs/Data/Injector.md

```diff
@@ -0,0 +1,52 @@
+## Module Data.Injector
+
+#### `Injector`
+
+``` purescript
+data Injector a b
+  = Injector (a -> b) (b -> Maybe a)
```

Comment:
Looks like it `purescript-profunctor` is indeed already included; if you like, I can look into making the switch.